deployments/gpu-operator/values.yaml

```diff
@@ -7,16 +7,52 @@ replicaCount: 1
 nfd:
   enabled: true
 
+node-feature-discovery:
+  worker:
+    tolerations:
+    - key: "node-role.kubernetes.io/master"
+      operator: "Equal"
+      value: ""
+      effect: "NoSchedule"
+    - key: "nvidia.com/gpu"
+      operator: "Equal"
+      value: "present"
+      effect: "NoSchedule"
+
+    options:
+      sources:
+        pci:
+          deviceLabelFields:
+          - vendor
+
 platform:
   openshift: false
 
 operator:
   repository: nvidia
   image: gpu-operator
-  tag: @OPERATOR_VERSION@
+  tag: 1.0.0
```

Comment:
Can you leave this one to @OPERATOR_VERSION@, we need to change the CI to set it correctly

Reply:
Is there a way to change this? (CI that update chart.yaml for example)
It's really hard to use it without releasing it as git.

Reply:
Yes but let's do this in a followup PR

Reply:
Yes of course :)
I update the PR on GitLab to remove the commit which change this.
(I anticipated your comment ;))